drivers/lora/sx1276.c

```diff
@@ -103,6 +113,10 @@ struct sx1276_data {
 #if DT_INST_NODE_HAS_PROP(0, rfo_enable_gpios) &&	\
 	DT_INST_NODE_HAS_PROP(0, pa_boost_enable_gpios)
 	u8_t tx_power;
+#endif
```

Comment:
Please don't add this condition in middle. Add it after the previous `#endif`

Reply:
I'm not sure what you mean here.
```
#if DT_INST_NODE_HAS_PROP(0, rfi_enable_gpios)
	struct device *rfi_enable;
#endif
#if DT_INST_NODE_HAS_PROP(0, rfo_enable_gpios)
	struct device *rfo_enable;
#endif
#if DT_INST_NODE_HAS_PROP(0, pa_boost_enable_gpios)
	struct device *pa_boost_enable;
#endif
#if DT_INST_NODE_HAS_PROP(0, rfo_enable_gpios) &&	\
	DT_INST_NODE_HAS_PROP(0, pa_boost_enable_gpios)
	u8_t tx_power;
#endif
#if DT_INST_NODE_HAS_PROP(0, tcxo_power_gpios)
	struct device *tcxo_power;
	bool tcxo_power_enabled;
#endif
```
This is the result after change that you commented. How would you change it?

Reply:
The resulting code looks OK but I was concerned about the diff. If you look into your diff, you'll notice that you've added `#endif` and then `#ifdef`.

Reply:
You need to look into `drivers: lora: sx1276: support TCXO power control` commit to understand what I'm referring to.

Reply:
Understood. This is how diff algorithm works. We don't have influence which `#endif` is added in diff :)